File: frontend/frontend_idea_app/lib/models/Event.dart
class Event {
  String date, timestamp, name;

  Event(this.date, this.timestamp, this.name);

  Event.fromJson(Map<String, dynamic> json)
      : name = json['name'],
        date = json['date'],
        timestamp = json['timestamp'];

  Map<String, dynamic> toJson() => {
        "name": "'" + name + "'",
        "date": "'" + date + "'",
        "timestamp": "'" + timestamp + "'"
      };
}

File: backend/ideappservice/src/main/java/com/ideapp/ideappservice/model/Event.java
package com.ideapp.ideappservice.model;

public class Event {
    String name;
    String timestamp;
    String date;

    public Event(String name,String timestamp,String date){
        this.name = name;
        this.timestamp = timestamp;
        this.date = date;
    }
    public Event(){}

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getName() {
        return name;
    }

    public String getTimestamp() {
        return timestamp;
    }

    public void setName(String eventName) {
        this.name = eventName;
    }

    public void setTimestamp(String time) {
        this.timestamp = time;
    }
}
